journalctl.js

```diff
@@ -12,8 +14,12 @@ function Journalctl (opts) {
 	if (opts.all) args.push('-a');
 	if (opts.lines) args.push('-n', opts.lines);
 	if (opts.since) args.push('-S', opts.since);
+	if (opts.until) args.push('-U', opts.until);
+	if (opts.reverse) args.push('-r');
+	if (opts.utc) args.push('--utc');
 	if (opts.identifier) args.push('-t', opts.identifier);
 	if (opts.unit) args.push('-u', opts.unit);
+	if (opts.output && supportedOutputFormats.includes(opts.output)) args.push('-o', opts.output);
```

Comment:
What is the reason for specifying the output format? The library always converts the stream to JS objects.